src/main/java/roomescape/auth/AuthConfig.java

```diff
@@ -0,0 +1,29 @@
+package roomescape.auth;
+
+import java.util.List;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import roomescape.infrastructure.LoginMemberArgumentResolver;
+
+@Configuration
+public class AuthConfig implements WebMvcConfigurer {
+    private final LoginMemberArgumentResolver loginMemberArgumentResolver;
+    private final AuthAdminInteceptor authAdminInteceptor;
+
+    public AuthConfig(LoginMemberArgumentResolver loginMemberArgumentResolver, AuthAdminInteceptor authAdminInteceptor) {
+        this.loginMemberArgumentResolver = loginMemberArgumentResolver;
+        this.authAdminInteceptor = authAdminInteceptor;
+    }
```

Comment:
`@Autowired` 어노테이션을 사용하여 의존성 주입을 명확하게 하는 것도 가독성에 좋을 거 같네요!
```suggestion
    @Autowired
    public AuthConfig(LoginMemberArgumentResolver loginMemberArgumentResolver, AuthAdminInteceptor authAdminInteceptor) {
        this.loginMemberArgumentResolver = loginMemberArgumentResolver;
        this.authAdminInteceptor = authAdminInteceptor;
    }
```